src/renderer/webpack.common.js

```diff
@@ -18,7 +18,7 @@ module.exports = {
   },
   resolve: {
     modules: [appPath, "node_modules"],
-    extensions: [".js", ".jsx", ".css"]
+    extensions: [".js", ".jsx", ".css", ".json"]
```

Comment:
This is required because the shapeshift.io module has an import like `const package = require('../package')

It causes the bundle to increase quite a bit. I will make a pr to that project to change it to `const package = require('../package.json')` which allow use to remove this